explorer/lib/explorer/periodically.ex

```diff
@@ -12,36 +12,38 @@ defmodule Explorer.Periodically do
   end
 
   def send_work() do
-    seconds = 12 # once per block
-    :timer.send_interval(seconds * 1000, :work) # send every n seconds
+    # once per block
+    seconds = 12
+    # send every n seconds
+    :timer.send_interval(seconds * 1000, :work)
   end
 
   def handle_info(:work, count) do
     # Reads and process last n blocks for new batches or batch changes
-      read_block_qty = 8
-      latest_block_number = AlignedLayerServiceManager.get_latest_block_number()
-      read_from_block = max(0, latest_block_number - read_block_qty)
+    read_block_qty = 8
+    latest_block_number = AlignedLayerServiceManager.get_latest_block_number()
+    read_from_block = max(0, latest_block_number - read_block_qty)
 
     Task.start(fn -> process_blocks_from_to(read_from_block, latest_block_number) end)
 
     # Gets previous unverified batches and checks if they were verified
-      run_every_n_iterations = 8
-      new_count = rem(count + 1, run_every_n_iterations)
-      if new_count == 0 do
-        Task.start(&process_unverified_batches/0)
-      end
+    run_every_n_iterations = 8
+    new_count = rem(count + 1, run_every_n_iterations)
+    # if new_count == 0 do
+    Task.start(&process_unverified_batches/0)
+    # end
```

Comment:
should this condition be checked? if not, let's remove the commented code

Reply:
yes, good catch